.coderabbit.yaml

```
@@ -0,0 +1,84 @@
+    ruff:
+      enabled: true
+    markdownlint:
+      enabled: true
+    github-checks:
+      enabled: true
+      timeout_ms: 90000
+    languagetool:
+      enabled: true
+      enabled_only: false
+      level: default
+      enabled_rules: [ ]
```

yamllint

[error] too many spaces inside empty brackets

(brackets)